release_notes.md

```diff
@@ -1,4 +1,24 @@
+## Release 1.5.2019 v2.1
+WAW was moved to Python 3.5 and from this release it is not compatible with Python 2. Last WAW version which supports python 2.7 is the 1.2.
+
+### New Features
+ - You can now delete, upload and test workspace by specifying its name or regular expression
+ - It is possible to replace some values using [workspace_addjson](/scripts/workspace_addjson.py) script
+ - There is now an option to set the log level in the CLI.
```

Comment:
Some bullets have period on the end and some not. They should be all same.

Reply:
Removed.